### PR TITLE
CTCTRADeRS-1308

### PR DESCRIPTION
**Pull request by kenneth-rayner**

### CODE REVIEW

File: app/controllers/addItems/DocumentTypeController.scala [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+    extends FrontendBaseController
+    with I18nSupport
+    with NunjucksSupport {
+
+  private val template = "addItems/documentType.njk"
+
+  def onPageLoad(lrn: LocalReferenceNumber, index: Index, documentIndex: Index, mode: Mode): Action[AnyContent] =
+    (identify andThen getData(lrn) andThen requireData).async {
+      implicit request =>
+        println("\n\n\n88888888888888888888\n\n\n\n")
```
Comment (PSouthern) [inline app/controllers/addItems/DocumentTypeController.scala:62]: Looks like some debugging here @kenneth-rayner

File: app/controllers/addItems/DocumentTypeController.scala [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+
+            val json = Json.obj(
+              "form"          -> preparedForm,
+              "index"         -> index.display,
+              "documentIndex" -> documentIndex.display,
+              "documents"     -> getDocumentsAsJson(preparedForm.value, documents.documentTypes),
+              "lrn"           -> lrn,
+              "mode"          -> mode
+            )
+            println("\n\n\n88888888888888888888\n\n\n\n")
```
Comment (PSouthern) [inline app/controllers/addItems/DocumentTypeController.scala:82]: Looks like some debugging here @kenneth-rayner

File: conf/messages.en [hunk 3]
```diff
@@ -1010,4 +1010,9 @@ documentExtraInformation.heading = What extra information do you need to add for
...
 documentExtraInformation.error.required = Enter the extra information for document {0}
 documentExtraInformation.error.length = Document extra information must be 26 characters or less
-documentExtraInformation.error.invalid = The extra information for this document must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, at signs (@), forward slashes, full stops, hyphens, percent signs, question marks, and less than and greater than signs
\ No newline at end of file
+documentExtraInformation.error.invalid = The extra information for this document must only include letters a to z, numbers 0 to 9, ampersands (&), apostrophes, at signs (@), forward slashes, full stops, hyphens, percent signs, question marks, and less than and greater than signs
+documentType.title = What type of document do you need to add?
+documentType.heading = What type of document do you need to add?
+documentType.checkYourAnswersLabel = Document type
+documentType.error.required = Enter the type of document you need to add
+documentType.error.length = DocumentType must be 100 characters or less
```
Comment (PSouthern) [inline conf/messages.en:1018]: Can we enforce this if the data is populated from reference data?

**Review by PSouthern — COMMENTED** [review]
@kenneth-rayner Couple of debug statements to remove and a question regarding the field input length to clarify.

File: conf/messages.en (hunk 3 above)
Comment (kenneth-rayner) [reply conf/messages.en]: No, removed it from messages and form provider spec

**Review by PSouthern — APPROVED** [review]

**Pull request merged**